src/Microsoft.ML.Console/Directory.Build.targets

```diff
@@ -0,0 +1,34 @@
+<Project>
+
+
+  <Target Name="CopyNativeAssemblies"
```

Comment:
You can move this up to a common `Directory.Build.targets` instead of duplicating it with the `test\Directory.Build.targets`.